manta-js/package/src/api/index.js

```diff
@@ -183,7 +183,7 @@ export default class Api {
         );
       }
       const pull_result = {
-        should_continue: result.should_continue,
+        should_continue: Boolean(result.should_continue),
```

Comment:
so what's the difference here actually ?

Reply:
![image](https://user-images.githubusercontent.com/110973578/221338020-50d3108d-3e69-440a-9689-e4ffcdf7ffd5.png)

So ```Boolean(result.should_continue) ``` is still a bug, it will always be true

it will be ```result.should_continue.isTrue```, that's works for me